frameworks/youth-risk-assessment/questions/current-behaviour.yml

```diff
@@ -0,0 +1,9 @@
+type: textarea
+question: How are they currently behaving in custody?
+hint: For example, how their behaviour may impact others or any behavioural needs they may have.
+description: Current behaviour
+prefill: false
+validations:
+  -
+    type: required
+    message: Enter details of the person’s current behaviour
```

Comment:
@JadeNConroy just a note that we can't add a flag to this question as its of type text